build.ps1

```diff
@@ -262,6 +262,14 @@ function cleanExtension([string] $bitness) {
             Remove-Item -Recurse -Force $_.FullName
         }
     }
+
+    if ($bitness -eq "64bit") {
+        $keepRuntimes = @('win', 'win-x64', 'win10-x64')
+    } elseif ($bitness -eq "32bit") {
+        $keepRuntimes = @('win', 'win-x86', 'win10-x86')
+    }
+    Get-ChildItem "$privateSiteExtensionPath\$bitness\workers\powershell\runtimes" -Exclude $keepRuntimes -ErrorAction SilentlyContinue |
+        Remove-Item -Recurse -Force -ErrorAction SilentlyContinue
```

Comment:
> Get-ChildItem "$privateSiteExtensionPath\$bitness\workers\node\grpc\src\node\extension_binary"

What about the `Get-ChildItem` call above in this function? Do we need `-EA` for that too?

Reply:
Yeah I can add that :)